workers/unity/Assets/Gdk/TransformSynchronization/Systems/LocalTransformSyncSystem.cs

```diff
@@ -1,6 +1,7 @@
 using Generated.Improbable;
 using Generated.Improbable.Transform;
 using Improbable.Gdk.Core;
+using Unity.Collections;
```

Comment:
Intentional?

Reply:
nope, will remove :)

Reply:
IF that was not intentional, why didn't the lint pick it up?

Reply:
good question! I didn't lint locally, but the TC lint also didn't pick it up